apm-agent-java/frameworks/wicket/WicketAjaxAPMFilter.java

```diff
@@ -0,0 +1,140 @@
+package eu.dindoffer.wicket.filter;
```

Comment:
maybe use `co.elastic.apm.contrib.wicket` as the package name?
```suggestion
package co.elastic.apm.contrib.wicket;
```